.github/workflows/ci.yml

```diff
@@ -232,6 +258,54 @@ jobs:
         password: ${{ secrets.PYPI_TOKEN }}
         verbose: true
 
+  coverage-report:
+    name: Coverage report
+    runs-on: ubuntu-latest
+    needs:
+      # We are waiting only for test jobs.
+      - test-linux
+      - test-windows
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Install dependencies
+      run: python -m pip install --upgrade pip coverage[toml] diff_cover
```

Comment:
Not much point in updating pip if you don't do that first and then make a second pip call to install the other stuff.  Both options are available below as suggestions.

(I just realized I could add multiple comments in the same discussion at the same time so that this can work...)

Reply:
```suggestion
      run: python -m pip install --upgrade coverage[toml] diff_cover
```

Reply:
```suggestion
      run: |
        python -m pip install --upgrade pip
        python -m pip install --upgrade coverage[toml] diff_cover
```